### DISCUSSION TITLE
a different prefix fflib2_

### DISCUSSION THREAD

**Discussion by PeterLin888** [wimvelzeboer/fflib-apex-extensions#10] [Q&A] [answered]
Hi William,
Would you consider using a different prefix fflib2_ ?
Thanks,
Peter

**Comment by wimvelzeboer** [comment, marked as answer]
The reason why I ended up with keeping the `fflib` prefix, is that I think some of the classes could (over time) be moved to the [fflib-apex-common](https://github.com/apex-enterprise-patterns/fflib-apex-common) repository.
To avoid tons of refactoring I thought about keeping the prefix the same.

Also I think the fflib2 prefix would be reserved for a real refactor of the mocks & common repos. Like a refactor that would break with backwards compatibility.

**Reply by PeterLin888** [reply]
Thanks William!

**Reply by PeterLin888** [reply]
This idea has been solved by #6 .
